hbase/Dockerfile

```diff
@@ -148,6 +166,7 @@ COPY --chown=stackable:stackable --from=builder /stackable/hbase-operator-tools-
 COPY --chown=stackable:stackable --from=builder /stackable/jmx /stackable/jmx/
 COPY --chown=stackable:stackable --from=builder /stackable/phoenix /stackable/phoenix/
 COPY --chown=stackable:stackable --from=builder /stackable/bin /stackable/bin/
+COPY --chown=stackable:stackable --from=builder /stackable/hadoop /stackable/hadoop/
```

Comment:
I think I read this correctly from above: This only copies the S3 jars and not _all of Hadoop_ right?
Because you already preselect them earlier?